content/intro-to-storybook/react/ko/data.md

```diff
@@ -1,152 +1,189 @@
 ---
 title: '데이터 연결하기'
 tocTitle: '데이터'
-description: 'UI 컴포넌트에 데이터를 연결하는 방법을 배워봅시다'
+description: 'UI 컴포넌트에 데이터를 연결하는 방법을 배워보세요'
 commit: '5da7c68'
 ---
 
-지금까지 우리는 독립된 환경에서 상태를 가지지 않는(stateless) 컴포넌트를 만들어보았습니다. 이는 Storybook에는 적합하지만 앱에 데이터를 제공하기 전까지는 유용하지 않습니다.
+지금까지 우리는 독립된 환경에서 상태를 가지지 않는(stateless) 컴포넌트를 만들어보았습니다. 이는 스토리북(Storybook)에는 적합하지만 앱에 데이터를 제공하기 전까지는 유용하지 않습니다.
 
 이번 튜토리얼에서는 앱 제작의 세부 사항에 중점을 두지 않기 때문에 자세히 설명하지 않을 것입니다. 그보다 컨테이너 컴포넌트(container components)에 데이터를 연결하는 일반적인 패턴을 살펴보도록 하겠습니다.
 
 ## 컨테이너 컴포넌트
 
-현재 작성된 `TaskList`는 구현됨에 있어서 외부와 어떠한 대화도 하지 않기 때문에 “표상적(presentational)”이라고 할 수 있습니다. ([이 블로그 포스트](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0)를 참조해주세요. 데이터를 얻기 위해서는 “컨테이너(container)”가 필요합니다.)
+현재 구현된 `TaskList`는 외부와 어떠한 소통도 하지 않기 때문에 “표상적(presentational)”이라고 할 수 있습니다. ([이 블로그 포스트](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0)를 참조해 주세요. 데이터를 얻기 위해서는 “컨테이너(container)”가 필요합니다.)
```

Comment:
@lisohh could you update this paragraph to match the English version [here](https://storybook.js.org/tutorials/intro-to-storybook/react/en/data/), to give you a bit more context on this, a while back I removed the reference to container/presentational components as technically it's an outdated pattern and actually not endorsed by the blog post author if you happen to check it out.

Reply:
I got it. Removed the blog link and slightly modified the next sentence to match the original context.